src/envo/envo-edit.owl

```diff
@@ -44006,6 +44007,14 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <htt
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_06105126> "weathering of plastic"@en)
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_06105126> <http://purl.obolibrary.org/obo/ENVO_06105119>)
 
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105265> (net shortwave radiation)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "http://disc.gsfc.nasa.gov/hydrology/data-holdings/parameters/net_shrtwave_rad.shtml") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_06105265> "A shortwave radiation where the quantity is the difference between incoming solar shortwave radiation and outgoing shortwave radiation from the earth surface."@en)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_06105265> <https://orcid.org/0000-0001-6284-4821>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_06105265> "2021-12-23T03:51:41Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_06105265> "net shortwave radiation"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_06105265> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002352> <http://purl.obolibrary.org/obo/ENVO_01001861>))
```

Comment:
you are missing an is-a parent

Apologies, we need to activate github actions so it will catch these

Reply:
Going to assume the parent class from @marieALaporte should suffice.